rbac/management/role/serializer.py

```diff
@@ -345,9 +345,14 @@ def obtain_groups_in(obj, request):
     qs = (
         assigned_groups
         | (
-            Group.platform_default_set().filter(tenant=request.tenant).filter(policies__in=policy_ids)
-            or Group.platform_default_set().filter(tenant=public_tenant).filter(policies__in=policy_ids)
-            or Group.admin_default_set().filter(tenant=public_tenant).filter(policies__in=policy_ids)
+            (
```

Comment:
This is a great catch ty for fixing it! 
Would it be possible to break up this nested union statement a bit to makes things a little more readable? I think we could have something like this:
```
platform_default_groups = <platform defaults for request tenant> or <platform defaults for public tenant>
admin_default_groups = <admin defaults for request tenant> or <admin defaults for public tenant>
qs = assigned_groups | platform_default_groups | admin_default_groups
...
```
imo that would make things a bit more clear and simpler to follow

Reply:
yes this is great idea.

Reply:
Good idea I definitely like the readability I will make those changes 👍